pkg/kfconfig/awsplugin/types.go

```diff
@@ -61,8 +62,9 @@ type Auth struct {
 }
 
 type BasicAuth struct {
-	Username string              `json:"username,omitempty"`
-	Password *kfconfig.SecretRef `json:"password,omitempty"`
+	Username string `json:"username,omitempty"`
+	//Password *kfconfig.SecretRef `json:"password,omitempty"`
+	Password string `json:"password,omitempty"`
```

Comment:
This has been changed to use String instead?

Reply:
I changed it to string because we only specify a hash of password in AWSPlugin.